cas/vaultcas/vaultcas.go

```diff
@@ -0,0 +1,354 @@
+package vaultcas
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/hex"
+	"encoding/json"
+	"encoding/pem"
+	"math/big"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
We should avoid using `github.com/pkg/errors` in new code, unless really required.

Reply:
Fixed with e29c85bb